common/src/main/java/gov/cms/ab2d/common/model/Contract.java

```diff
@@ -63,7 +67,7 @@ public Contract(@NotNull String contractNumber, String contractName, Long hpmsPa
     private Set<CoveragePeriod> coveragePeriods = new HashSet<>();
 
     public boolean isTestContract() {
-        return updateMode == UpdateMode.TEST;
+        return contractType == ContractType.OLD_TEST || contractType == ContractType.SYNTHEA;
```

Comment:
This is ok.

A fancier suggestion is to delegate to the enum as to whether it is a TEST.  e.g. return contractType.isTest();